checker/src/main/java/org/checkerframework/checker/formatter/FormatterVisitor.java

```diff
@@ -88,9 +89,14 @@ public Void visitMethodInvocation(MethodInvocationTree node, Void p) {
                                     default:
                                         if (!fc.isValidParameter(formatCat, paramType)) {
                                             // II.3
+                                            ExecutableElement method =
```

Comment:
I would probably move this outside of the loop, to make clear that it doesn't change each iteration.

Reply:
It's within the loop because it won't every be used in most loop executions.  Executing it outside the loop is only faster if on average every loop uses it more than 1 time, but is wasteful if each loop on average uses it less than 1 time.